helm_deploy/nomis-user-roles-api/values.yaml

```diff
@@ -35,6 +35,8 @@ generic-service:
       JDBC_URL: "JDBC_URL"
       DB_USERNAME: "DB_USERNAME"
       DB_PASSWORD: "DB_PASSWORD"
+      REPORTING_CLIENT_ID: "REPORTING_CLIENT_ID"
+      REPORTING_CLIENT_SECRET: "REPORTING_CLIENT_SECRET"
 
```

Comment:
These secrets will be added manually to deployment secrets since they will be supplied by NART not HMPPS-Auth, they are *not* OAuth2 credentials